src/app/components/Metadata/index.jsx

```diff
@@ -172,9 +172,9 @@ Metadata.propTypes = {
     propTypeCheck(props, propName, 'Metadata', arrayOf(string)),
   themeColor: string.isRequired,
   timeFirstPublished: (props, propName) =>
-    propTypeCheck(props, propName, 'Metadata', number.isRequired),
+    propTypeCheck(props, propName, 'Metadata', string.isRequired),
```

Comment:
What exactly does this do? Can you find out if it adds any value? I don't think there should be abstractions over prop types. Prop types should be as explicit as possible.

Reply:
I agree, opening another issue to investigate why we are doing this would be a good idea.